packages/i18n/src/sprintf.js

```diff
@@ -28,8 +28,9 @@ export function sprintf( format, ...args ) {
 	try {
 		return sprintfjs.sprintf( format, ...args );
 	} catch ( error ) {
-		logErrorOnce( 'sprintf error: \n\n' + error.toString() );
-
+		if ( error instanceof Error ) {
```

Comment:
This one is good, `sprintfjs` throws only `Error` instances:

https://unpkg.com/sprintfjs@1.2.17/sprintf.js